fetchai/ledger/api/__init__.py

```diff
@@ -63,9 +69,12 @@ def __init__(self, host=None, port=None, network=None):
     def sync(self, txs: Transactions, timeout=None):
         timeout = int(timeout or 120)
         # given the inputs make sure that we correctly for the input set of values
+        finished = []
         if isinstance(txs, str):
+            initial_txs = {txs}
```

Comment:
I am not sure this is used - is that correct?

Reply:
Done.